docs/source/conf.py

```diff
@@ -31,6 +31,21 @@
 # https://sphinx-rtd-theme.readthedocs.io/en/stable/installing.html
 import sphinx_rtd_theme  # noqa
 
+
+def download_with_headers(url, filename):
+    """Download a file with proper User-Agent header to avoid 403 errors."""
+    try:
+        req = urllib.request.Request(
+            url, headers={"User-Agent": "ImpactX-docs-builder"}
```

Comment:
Fix User-Agent value:
```suggestion
            url, headers={"User-Agent": "pyAMReX-docs-builder"}
```